Final_Project_Billinis-AR.qmd

```diff
@@ -0,0 +1,132 @@
+---
+title: "Final_Project_GRShipping"
+author: "Alex Billinis"
+format: html
+editor: visual
+---
+
+## Quarto
+
+Quarto enables you to weave together content and executable code into a finished document. To learn more about Quarto see <https://quarto.org>.
+
+## 
+
+```{r}
+library(ggplot2)
```

Comment:
So one of the things to think about, and you can look at the Journal of Digital History for an example of this, is how to balance the historical narrative in a digital paper such as this with the actual code. One thing you might consider to keep the attention on the narrative and the images that support your argument is to set blocks that are essential to your argument so that they run in the background. You can do that if you add ```#| echo: false``` to the top of the block.